integration-tests/application-server-integration-tests/pom.xml

```diff
@@ -162,7 +162,12 @@
             <version>1.12.1</version>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
```

Comment:
This is an explanation why you use it as a `test-jar` dependency rather than a `test` dependency?

Reply:
No, in fact for this module (and also in others) forgot to set the `test` scope to this `test-jar` artifact, I'll fix that. What I meant here with Jetty is that having this dependency included in any application that is deployed on Jetty will fail, so for example it would be a bad idea to add it to all agent plugins modules and sub-modules as it would just trigger this Jetty issue.

Reply:
I see, but it is not special to this one, is it? We don't expect a test app to have a dependency on anything else from the project in general I assume.

> in fact for this module (and also in others) forgot to set the test scope to this test-jar artifact

I don't think it matters in this case